Codecademy/gamut
# style(BookAndNotepad): Replace current svg with new cropped svg

### Overview


The current illustration for the BookAndNotepad has extra space on the right side making it hard to center (see pics). Once we have the updated illustration, replace the current BookAndNotepad svg in [gamut illustrations](https://gamut.codecademy.com/?path=/docs/atoms-illustrations--editable-illustration) and then bump the monolith package so it gets the corrected illustration.

**Before** (note the extra space on the right)
<img width="482" alt="Screenshot 2023-07-12 at 12 34 20 PM" src="https://github.com/Codecademy/gamut/assets/12686946/0df52602-5038-4da9-ae9c-0bd50a7f984b">

**After**
<img width="463" alt="Screenshot 2023-07-12 at 12 33 59 PM" src="https://github.com/Codecademy/gamut/assets/12686946/195cba0c-7e56-4ff4-b34d-7e622e252fbd">



### PR Checklist

- [x] Related to designs: https://www.figma.com/file/TjBVb65kukF5TZRqYUn4Se/onboarding?node-id=306-11550
- [x] Related to JIRA ticket: [DISC-1226]

#### Testing Instructions



1. Check out the BookAndNotepad under Illustrations in Storybook. You can see its more centered than the one in production which has extra space on the right. 
2. You can also go on the monolith PR env link, go to the dashboard as a user in the `onboarding_getting_started` experiment `variant`, and see the new illustration in action

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/36035)  | [Monolith Env](https://pr-36035-monolith.dev-eks.codecademy.com/) |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/2697)  |     |





[DISC-1226]: https://codecademy.atlassian.net/browse/DISC-1226?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
